snapcraft/storeapi/errors.py

```diff
@@ -369,6 +370,22 @@ def __init__(self, result):
         super().__init__()
 
 
+class StoreServerError(StoreError):
+
+    fmt = '{what}: {error_text} (code {error_code}).\n{action}'
+
+    def __init__(self, response):
+        what = 'The store encountered a server error'
```

Comment:
“The Snap Store encountered an error while processing your request:”
…
“The operational status of the Snap Store can be checked at https://status.snapcraft.io”

Let’s be consistent. Snap Store is always capitalised.

Reply:
Alright, done.